frontend/src/components/EntityBulkAdd.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div v-for="num in entityFormComponentsCountRange" :key="`ef-${num}`">
+    <EntityForm :ref="`entity-form-${num}`" :schema="schema"
+                :attributes="attributes" :batch-mode="true" @save-all="saveAll"/>
+  </div>
+  <div class="container mt-2">
+    <button class="btn btn-outline-secondary w-100" @click="addEntityForm">
+      <i class='eos-icons'>add_circle</i>
+      Add more
+    </button>
+  </div>
+</template>
+
+<script>
```

Comment:
could have been a `<script setup>` :cry: :stuck_out_tongue:

Reply:
Nice suggestion ;) We should do this across the app, since I didn't found any occurrences.. do you think that this deserves to be a new issue?

Reply:
I don't think so.. I wouldn't like to dedicate extra effort to refactor components just because I like the style more. I still think it is going to make the code better - so (if it was up to me :stuck_out_tongue: ) I'd like us to slowly transition - i.e. if someone writes a new component from scratch: composition api (hence the comment), or if someone does some extended component refactoring that might be simplified with composition api: go for it. But let's not rewrite components 'just because' - it might not be the best usage of the team's time.

Reply:
...and don't feel pressured, I understand that it's something new that requires a bit extra effort to ease into, so this was just a tiny, friendly nudge :slightly_smiling_face:

Reply:
Ah, I see what you mean now.. You got a point. I will try to do it, It will be nice exercise and reasonable investment :)